### ISSUE TITLE
[Functional-Social Embedded- Profile]: "Navigate Up" button functionality is not working.

### ISSUE THREAD

**Issue by billycamp** [Microsoft/EmbeddedSocial-Android-SDK#29]
Priority: 

Repro Steps
1. Install ""One Bus Away"" app on Android.  
2. Start ""One Bus Away"" app from Android phone.
3. ""One Bus Away "" Home page will get appear.
4. Navigate to ""Open Navigation Drawer"" button and press ENTER (or double tap) to open.
5. Navigate to ""Sign In"" and press ENTER (or double tap) to open. 
6. Navigate to ""Sign In with google"" and press ENTER (or double tap) to open.
7. Login with valid google account credentials.
8. “Create Profile” page will get appear.
9. Fill all the input fields with valid inputs.
10. Navigate to ""Done"" button and press ENTER.
11. Navigate to ""Open Navigation Drawer"" button and press ENTER  (or double tap).
12. Navigate to ""My Profile"" and press ENTER  (or double tap) to open.
13. Navigate to  ""New Follow Requests"" and press ENTER  (or double tap) to open.
14. Verify that ""Navigate Up"" button is working or not.

Expected
""Navigate Up"" button functionality should be work properly so that user can come back previous page using ""Navigate Up"" button.

Actual
""Navigate Up"" button functionality is not working. User unable to come back previous page using ""Navigate Up"" button.

MAS Rule: Functional

Impact
User would get confuse and face difficulty  If the ""Navigate Up"" button functionality is not working.

Test Environment
OS: Android
Device Model No. : Nexus 5
Android Version : 6.0.1
App: Social Embedded
App Version : v2.2.3 (84)
Tool : Talk Back
